src/rsz/src/RepairDesign.cc

```diff
@@ -1385,4 +1404,35 @@ RepairDesign::metersToDbu(double dist) const
   return dist * dbu_ * 1e+6;
 }
 
+void
+RepairDesign::printProgress(int iteration, bool force, bool end, int repaired_net_count) const
+{
+  const bool start = iteration == 0;
+
+  if (start && !end) {
+    logger_->report("Iteration | Resized | Buffers | Nets repaired | Left to check");
```

Comment:
Maybe "nets remaining" instead of "left to check"?

Reply:
I thought about doing that, but it's not actually nets, it's `Vertex`'s in OpenSTA ,so there are usually more than the number of nets. If you have a better suggestion for the phrasing I'm open since it's not great as it is.

Reply:
Maybe just "remaining" or "checks remaining"? It's fine either way I just like the formal-sounding term.

Reply:
I'll make it remaining